modules/auto-lock-process-baselines-known-limitations.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_mod-docs-content-type: CONCEPT
+[id="auto-lock-process-baselines-known-limitations_{context}"]
+= Auto-lock process baselines known limitations
+
+[role="_abstract"]
+Central, Central DB, and Sensor consume more CPU and memory resources when process baseline auto-locking is enabled. This can lead to CPU throttling and pods crashing due to running out of memory.
+
+The following results were obtained from tests with 1,000 deployments in which 5,000 process were spawned every 30 seconds (166.67 processes per second). The test was run with the feature enabled and disabled. Resource usage was compared between the two tests. For the tests the process baseline generation duration was set to three minutes and the rate of process creation did not change after the baseline generation period ended.
+
+* Sensor used 24 Mb more memory.
+* The difference in sensor memory usage didn’t seem to be increasing with time.
```

Comment:
OK, if we are leaving these in here, there are a couple of changes I would suggest.


```suggestion
* The difference in sensor memory usage apparently did not appear to increase over time.
```